README.md

```diff
@@ -425,6 +425,33 @@ Sometime it is required to load sensitive configurations (GitHub/Snyk's token) f
 * Change the workdir of the docker image to be `/broker`/
 Example of such file is located in your broker container at $HOME/.env
 
+
+
+### Troubleshooting
+
+#### Fail to create Fix PR in GitHub / GitHub Enterprise
+
+One of the reason might be failing of fetching big manifest files (>1 Mb). To address this issue, few additional endpoints should be whitelisted in `accept.json`:
```

Comment:
This is not only about fix PRs, also about tests. Not being able to fetch a big lock file has many effects.

I'd name this section something like `Need to support files bigger than 1MB in Github / Github enterprise?`

Also, we need to explain why is this not in the default accept.json: because allowing the blob API lets Snyk access any file within the repo.


By the way, the file sha API is not a security problem so we should add it to the default accept.json - less rules to add manually